standard/standard/annex-ats.adoc

```diff
@@ -1,1085 +1,1279 @@
-
-Although implementation of the Standardization Targets, expressed as SDUs, is independent of the Logical Model, GeoPose 1.0 also defines one of many possible implementations, single encoding in JavaScript Object Notation (JSON). The encodings of the eight targets are specified using JSON-Schema-2019-09. To keep the individual Standardization targets independent, there are some SDU requirements and corresponding conformance tests that appear in more than one requirement or conformance class. This structure is based on the judgement that it is easier to understand the independence of targets with complete definitions than would be the case if the definitional requirements of the SDUs were factored out and referenced indirectly by individual encodings.
+GeoPose 1.0 specifies a eight Standardization Targets using elements of the Logical
+Model. These elements are Structural Data Units (SDUs) and they have the stereotype
+"Structural Data Unit - SDU". Each SDU is an element of the Logical Model that will
+be expressed in concrete data objects encoded using specific encoding or
+serialization technologies.
+
+Although implementation of the Standardization Targets, expressed as SDUs, is
+independent of the Logical Model, GeoPose 1.0 also defines one of many possible
+implementations, single encoding in JavaScript Object Notation (JSON). The encodings
+of the eight targets are specified using JSON-Schema-2019-09. To keep the individual
+Standardization targets independent, there are some SDU requirements and
+corresponding conformance tests that appear in more than one requirement or
+conformance class. This structure is based on the judgement that it is easier to
+understand the independence of targets with complete definitions than would be the
+case if the definitional requirements of the SDUs were factored out and referenced
+indirectly by individual encodings.
 
 === Global Conformance Class
 
 Conformance with the Global Requirements is required for all implementations.
 
 [[conf_global]]
-[width="100%",%header,cols="2,6"]
-|===
-<|*Conformance Class:* <|*Global* 
-
-<s|ID |<<conf_global,/conf/global>>
-<s|Target Type|SDU
-<s|Description| Conformance with Global SDU requirements
-<s|Dependency| None
-<s|Test: G-Target|/conf/target_dependency
-<s|Test: G-SDU|/conf/sdu_general 
-<s|Test: G-LM|/conf/lm_general 
-
-|===
+[conformance_class]
+====
+[%metadata]
+label:: <<conf_global,/conf/global>>
+subject:: <<rc_global,Requirement Class "Global">>
+classification:: Target Type:SDU
+description:: Conformance with Global SDU requirements
+inherit:: None
+
+[conformance_test,id="rta1"]
+======
+[%metadata]
+label:: <<conformance_global,/conf/target_dependency>>
```

Comment:
A 'label' is a string, an identifier. Why is it a cross-reference?

Reply:
That is not allowed in the Metanorma ModSpec format. The existence of these links must have been a mistake because there is no point in referencing the table that this requirement is in. Please help change these into just the textual identifier, i.e. here `/conf/global`.

Reply:
Ok, thanks!